src/getPublicKeys.c

```diff
@@ -228,7 +275,7 @@ void getPublicKeys_handleGetNextKeyAPDU(
 	parsePath(&view);
 	VALIDATE(view_remainingSize(&view) == 0, ERR_INVALID_DATA);
 
-	returnOneKey();
+	promptAndRespondOneKey();
```

Comment:
the "prompt" verb may be imho misleading given that the prompt is triggered only under certain condiditons. Imho a name like  `getPublicKeyFlow()` would be more appropriate as it gives the information to the reader that it will get one public key and some user-interaction may or may not be happening as it is a flow